sections/codestylepractices/eslint_prettier.german.md

```diff
@@ -0,0 +1,26 @@
+# Verwendung von ESLint und Prettier
+
+
+### ESLint und Prettier im Vergleich
+
+Wenn Du diesen Code mit ESLint formatierst, erhältst Du lediglich eine Warnung, dass er zu breit ist (abhängig von der `max-len` Einstellung). Prettier formatiert es automatisch für Dich.
```

Comment:
Maybe I'd use a lowercase "Du" here, although both forms are possible I'd prefer the lowercased standard

Reply:
Will do! Will probably fix it in a second commit, along with another translated piece